src/error.rs

```diff
@@ -95,19 +62,26 @@ pub enum UfoLoadError {
     LibFileMustBeDictionary,
     /// Failed to load a file from the data store.
     #[error("failed to load data store")]
-    LoadingDataStore(#[source] StoreEntryError),
+    LoadDataStore(#[source] StoreEntryError),
```

Comment:
we can also get rid of `Load` in these variants, since we're in a `FontLoadError`?